.github/workflows/hardhat-tasks.yml

```diff
@@ -60,6 +60,19 @@ jobs:
           pnpm deploy:localhost
           pnpm deploy-poll:localhost
           pnpm initPoll:localhost --poll 0
+
+          sleep 30
+
+          pnpm run merge:localhost --poll 0
+
+          pnpm run prove:localhost --poll 0 \
+              --coordinator-private-key "macisk.1751146b59d32e3c0d7426de411218172428263f93b2fc4d981c036047a4d8c0" \
```

Comment:
Can we use secrets for maci private key later?

Reply:
for now actually reverting this part as we'll need to add some recipients otherwise call to add results will fail